Provisioning/Vault/audit.tf

```diff
@@ -0,0 +1,7 @@
+resource "vault_audit" "file" {
+  type = "file"
+
+  options = {
+    file_path = "stdout"
```

Comment:
wanted to see the certificates issued so I enabled the audit log to container stdout